README.md

```diff
@@ -1,27 +1,25 @@
-# The Rust Programming Language
-
-[![Rust Community](https://img.shields.io/badge/Rust_Community%20-Join_us-brightgreen?style=plastic&logo=rust)](https://www.rust-lang.org/community)
+<p align="center">
+  <img alt="The Rust Programming Language: A language empowering everyone to build reliable and efficient software" src="https://www.rust-lang.org/static/images/rust-social-wide.jpg" width="512" height="256">
+  <br>
+  <a href="https://www.rust-lang.org/">Website</a> |
+  <a href="https://www.rust-lang.org/learn/get-started">Getting started</a> |
+  <a href="https://www.rust-lang.org/learn">Learn</a> |
+  <a href="https://www.rust-lang.org/learn#learn-use">Documentation</a> |
+  <a href="CONTRIBUTING.md">Contributing</a>
```

Comment:
These can continue to be normal Markdown links and thus all as one line, inline, with the links provided in Markdown "footer" later, making it clear how it should look. As long as you use line breaks appropriately between things, you can intermingle HTML and Markdown syntax in a CommonMark or GitHub-Flavored Markdown document (because GHF strives to be compatible with CommonMark). So just use a line break before and after the Markdown syntax. The HTML will still be included directly and will still affect the Markdown after.

Reply:
Done.